stories/action-sheet/action-sheet.stories.js

```diff
@@ -0,0 +1,199 @@
+import '../../dist/avatar.css';
+import '../../dist/icon.css';
+import '../../dist/action-sheet.css';
+
+export default {
+    title: 'Components/Action Sheet',
+    parameters: {
+        description: `
+An action sheet consists of a list of options a user can select from to complete an action. Actions can be
+clustered if there is not enough space on the screen.
+            
+## Usage
+
+**Use the action sheet if:**
+
+- You need an option that provides more than one action.
+- It is really important that the user stays in context on a phone.
+- You only have a small number of actions.
+
+**Do not use the action sheet if:**
+
+- The menu provides only one option. In this case, consider using a **Button** instead.
+- You need to show a hierarchical menu. In this case, use the **MenuButton** instead.
```

Comment:
Button is the only one that should be emphasized, since it's the main component. However, button and split button are technically components, in the documentation they are considered component "types" / or rather button types. They don't need to be emphasized, you can type them out like you did before :)

"menu button" "split button"